distributions/aws/OWNERS

```diff
@@ -1,4 +1,5 @@
 approvers:
-  - jeffwan
-  - mameshin
-  - PatrickXYS
+  - rrrkharse
+  - surakota
```

Comment:
my github username is surajkota

lets remove this file anyway because we don't have infra which can act to it

Reply:
I think we should keep this file for ownership pov. What do you mean by infra acting on it. ? notifying on PR ?